hack/run-e2e-conformance-virtual-cluster.sh

```diff
@@ -105,6 +105,43 @@ kcli create cluster generic --paramfile ./${cluster_name}-plan.yaml $cluster_nam
 export KUBECONFIG=$HOME/.kcli/clusters/$cluster_name/auth/kubeconfig
 export PATH=$PWD:$PATH
 
+# w/a switch flannel images to use quay repo to avoid dockerhub pull limit
+# [SchSeba] there is a cronjob in place to make a copy of the image between dockerhub and quay account
+ds=`kubectl -n kube-flannel get daemonset kube-flannel-ds -oyaml`
```

Comment:
we already have a local registry set right ?

i wonder if its better to build our own flannel image (make image) from a given release tag of the repo.
then reference it in the flannel ds and deploy. that way we dont rely on private repo.

WDYT?

Reply:
I think I can try that I will need to check the flannel code